src/app/pages/apps/store/available-apps-store.service.ts

```diff
@@ -0,0 +1,289 @@
+import { Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+import { TranslateService } from '@ngx-translate/core';
+import _ from 'lodash';
+import {
+  Observable, catchError, combineLatest, map, of, switchMap, tap,
+} from 'rxjs';
+import { AppExtraCategory } from 'app/enums/app-extra-category.enum';
+import { AppsFiltersSort, AppsFiltersValues } from 'app/interfaces/apps-filters-values.interface';
+import { AvailableApp } from 'app/interfaces/available-app.interfase';
+import { WebsocketError } from 'app/interfaces/websocket-error.interface';
+import { ApplicationsService } from 'app/pages/apps/services/applications.service';
+import { DialogService } from 'app/services';
+import { ErrorHandlerService } from 'app/services/error-handler.service';
+
+export interface AppsByCategory {
+  title: string;
+  apps: AvailableApp[];
+  totalApps: number;
+  category: string;
+}
+
+export interface AvailableAppsState {
+  availableApps: AvailableApp[];
+  latestApps: AvailableApp[];
+  recommendedApps: AvailableApp[];
+  categories: string[];
+  appsByCategories: AppsByCategory[];
+  filteredApps: AvailableApp[];
+  filter: AppsFiltersValues;
+  isLoading: boolean;
+  isFilterApplied: boolean;
+  appsPerCategory: 6;
```

Comment:
Looks like it needs to be `number`.

Reply:
I left the type as a specific digit as a way of making this value constant so it can't be set to any other number while being used in the store service. If this value needs to change, this type will also need to be changed.